### PR TITLE
Reader: remove Flux-powered follow button

### PR DESCRIPTION
**Pull request by bluefuton**
With the release of Following Manage, all follows are now powered by Redux state rather than Flux stores. The feature flag `reader/following-manage-refresh` has already made the switch.

This PR removes the now-defunct Flux version of the button.

### CODE REVIEW

File: client/blocks/follow-button/index.jsx [hunk 1]
```diff
@@ -1,71 +1,52 @@
 /**
```
Comment (bluefuton) [inline client/blocks/follow-button/index.jsx]: This is copied from `client/blocks/follow-button/redux.jsx` (no changes).

File: client/reader/follow-button/index.jsx [hunk 2]
```diff
@@ -1,12 +1,44 @@
 /**
```
Comment (bluefuton) [inline client/reader/follow-button/index.jsx]: This is copied from `client/reader/follow-button/redux.jsx` (no changes).

**Comment by samouri** [comment]
code lgtm, testing now

**Comment by samouri** [comment]
does it build for you?

screen shot 2017-05-25 at 10 12 44 am

**Comment by bluefuton** [comment]
Thanks for fixing the busted include @samouri 👍

**Pull request merged**